spec/controllers/v0/users_controller_spec.rb

```diff
@@ -56,6 +56,8 @@
 
     before do
       sign_in_as(user)
+      Flipper.disable(:va_v3_contact_information_service)
+      Flipper.disable(:remove_pciu)
       Flipper.disable(:profile_user_claims)
```

Comment:
Could you mock these? Enabling/disabling Flippers affects the global state, which can lead to unintended side effects in other tests, especially when running in parallel. Mocking will help keep things isolated e.g.
```ruby
allow(Flipper.va_v3_contact_information_service).to receive(:enabled?).and_return(false)
```

Reply:
I can definitely mock them. I was disabling them to make sure all areas are addressed. I will definitely mock them in the next pull request.

Reply:
@rileyanderson This is addressed in another PR.  I will remove these lines.

Reply:
@rileyanderson I attempted to do this but since contact information is used throughout vets-api, this has caused many issue with testing. These issues should be worked out but it is not in scope with this current issue. The flipper should be removed in a few weeks.

Reply:
Oh I see, there was already a discussion about the Feature flipper mocking vs setting it in here. So you're saying the specs work when the Feature flipper is directly set here, but they do not work when they are mocked using `allow(Flipper.va...).to receive(enabled?).and_return(...)`?

Reply:
Because in that case I believe this is proof of spec pollution and very likely to cause flaky specs

Reply:
What I'm trying to say is whatever other file doesn't work when you do `allow(Flipper.va...).to receive(enabled?).and_return(...)`, it's not working because it expects Flipper to be a certain value and it isn't that value since you only mocked it in this file.

However, if left as is, if you run specs and that other file is run before this spec is run, then it will also fail. But now the failure is order dependent. Whereas with the Flipper mocking you at least know where the spec failure is directly